packages/ui/client/components/views/ViewEditor.vue

```diff
@@ -24,7 +25,9 @@ watch(
       draft.value = false
       return
     }
-    code.value = (await client.rpc.readTestFile(props.file.filepath)) || ''
+    code.value = isReport
+      ? ('source' in props.file ? props.file.source : '') as string
+      : (await client.rpc.readTestFile(props.file.filepath)) || ''
```

Comment:
Let's not make our lives harder by having `isReport` everywhere. Just make `readTestFile` return the source